### DISCUSSION TITLE
List View For Files and URLs Tabs

### DISCUSSION THREAD

**Discussion by spencerhooks** [diced/zipline#302] [Ideas]
It would be great to have a list view (like is already in the files area of the home tab) available for the Files and URLs tabs. This could be in addition to the existing gallery view and could be user selectable. I think it especially makes sense for the URLs tab, where there aren't images to show in the gallery view.

**Comment by diced** [comment]
Seems like a great idea, I will think about implementing this.

**Comment by diced** [comment]
Added in d1a486ac1f1a0a4469e39c8fe4aa2af56ee49a96
